packages/core/src/components/checkbox/checkbox.scss

```diff
@@ -1,11 +1,11 @@
 @use '../../helpers';
+@use '../../internal';
 @use '../../mixins';
-@use '../input-container/input-container' as components;
 
 @mixin Checkbox() {
   $stroke-size: helpers.space(0.25);
 
-  @include components.InputContainer();
+  @include internal.IndicatorContainer();
```

Comment:
We have to check whether including both Checkbox and Radio is going to repeat CSS from Indicator.

If it does, we probably shouldn't bundle, but instead document they're coupled

Reply:
Yes, it does compile the indicator container twice...

Reply:
What do you think, if internals would be included in components by default, but could also be excluded:

```scss
@mixin Component($include-internal: true) {
  @if $include-internal {
    @include internal.SomethingInternal();
  }
}
```

when want to compile all:

```scss
@mixin components() {
  $include-internal: false;

  @include castor.Internal();
  @include castor.Component($include-internal);
}
```

Docs should also clearly indicate such possibility, however then internals must be exported by default.

Reply:
That might be confusing in this very common scenario

```scss
:root {
  @include castor.Checkbox();
  @include castor.Radio($include-indicator: false);
}
```

This feels somewhat more natural, but also there's no telling they're coupled.

```scss
:root {
  @include castor.Checkbox();
  @include castor.Radio();
  @include castor.Indicator();
}
```

Another option is to include indicator in both, which allows them to be included separately.
Then when both are needed:

```scss
:root {
  @include castor.CheckboxAndRadio(); // comes with Indicator
}
```

Don't like that much though, doesn't scale well
`castor.CheckboxAndRadioAndInputAndTextarea()` 🤮 

Lastly, we could not export individual mixins and instead have logic in `components`:

```scss
@include castor.components((
  'button',
  'checkbox',
  'input',
  'radio',
));

@mixin components($list: ()) {
  @if 'Checkbox' in $list {
    @include castor.Checkbox();
  }

  @if 'Checkbox or 'Radio' in $list {
    @include castor.Indicator();
  }
}
```

The downside is loose strings instead of tight mixin names.

Reply:
I've changed this part now to track what additions are being included, so such are only included once per build.